README.md

```diff
@@ -36,13 +37,50 @@ Supports most Debian and RHEL-based Linux distributions, and Windows.
+------------------------------|------------------------|
+6                             | https://s3.amazonaws.com/ddagent-windows-stable/datadog-agent-6-latest.amd64.msi |
+7                             | https://s3.amazonaws.com/ddagent-windows-stable/datadog-agent-7-latest.amd64.msi |
+
+To override the default behavior, set the `datadog_windows_download_url` variable to something else than an empty string.
+
 ## Role Variables
 
 | Variable                                                                                                                                        | Description                                                                                                                                                                                                                                              |
 |-------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `datadog_api_key`                                                                                                                               | Your Datadog API key.                                                                                                                                                                                                                                    |
 | `datadog_site`                                                                                                                                  | The site of the Datadog intake to send Agent data to. Defaults to `datadoghq.com`, set to `datadoghq.eu` to send data to the EU site. This option is only available with agent version >= 6.6.0.                                                         |
-| `datadog_agent_version`                                                                                                                         | The pinned version of the Agent to install (optional, but highly recommended). Examples: `1:6.0.0-1` on apt-based platforms, `6.0.0-1` on yum-based platforms, `6.0.0` on Windows platforms. **Note** Downgrades are not supported on Windows platforms. |
+| `datadog_agent_major_version`                                                                                                                         | The major version of the Agent which will be installed. Its possible values are 5, 6, 7. If both this and `datadog_agent_version` are not set, then `datadog_agent_major_version` defaults to 7. If this is not set but `datadog_agent_version` is set, then `datadog_agent_major_version` is set to the major version provided in `datadog_agent_version`. |
```

Comment:
```suggestion
| `datadog_agent_major_version`                                                                                                                         | The major version of the Agent which will be installed. Its possible values are 5, 6 or 7 (default to 7). If set `datadog_agent_version` will take precedence, otherwise the latest version of the specified major will be installed. Setting `datadog_agent_major_version` is not needed if `datadog_agent_version` is used. |
```